code/modules/admin/verbs/adminhelp.dm

```diff
@@ -671,8 +671,10 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		if("reject")
 			Reject()
 		if("reply")
-			HandleIssue() /// SKYRAT EDIT ADDITION - ADMIN HANDLE
-			usr.client.cmd_ahelp_reply(initiator)
+			// SKYRAT EDIT START - ADMIN HANDLE
+			if(handle_issue())
+				usr.client.cmd_ahelp_reply(initiator)
```

Comment:
```suggestion
				usr?.client.cmd_ahelp_reply(initiator)
```

Reply:
Technically speaking, this is upstream code that I just moved, but yeah I guess I can do that too.